crates/rpc/rpc/src/eth/error.rs

```diff
@@ -314,6 +324,14 @@ pub enum RpcInvalidTransactionError {
     /// Blob transaction is a create transaction
     #[error("blob transaction is a create transaction")]
     BlobTransactionIsCreate,
+    /// A deposit transaction was submitted as a system transaction post-regolith.
+    #[error("no system transactions allowed after regolith")]
+    #[cfg(feature = "optimism")]
+    DepositSystemTxPostRegolith,
+    /// A deposit transaction halted post-regolith
+    #[error("deposit transaction halted after regolith")]
+    #[cfg(feature = "optimism")]
+    HaltedDepositPostRegolith,
```

Comment:
same pattern

Reply:
addressed in https://github.com/anton-rs/op-reth/pull/132

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/140